packages/mjml-carousel/README.md

```diff
@@ -58,7 +57,6 @@ css-class | string | class name, added to the root HTML element created | n/a
 href | url | link to redirect to on click | n/a
 rel | string | specify the rel attribute | n/a
 src | url | image source | n/a
-target | string | link target on click | \_blank
```

Comment:
Looks like to be used in the carouselImage, this should be in allowedAttributes instead